kubernetes/deploy_k8s.yml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eleuther-neox
+spec:
+  strategy:
+    type: Recreate
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: eleuther-neox
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: eleuther-neox
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: neox
+        command: ["/usr/sbin/sshd"]
+        args: ["-D"]
+        tty: true
+        image: leogao2/deepspeed_eleuther
+        ports:
+          - name: sshd
+            containerPort: 2222
```

Comment:
shouldn't this be `22` as suggested by the `EXPOSE` at https://github.com/coreweave/cuda-ssh-server/blob/master/Dockerfile#L37

Reply:
That's what I had thought as well, but Leo and I tested it and it seems to work with 2222. Not sure why / what difference it makes though.